### ISSUE TITLE
Hook to get the latitude and longitude

### ISSUE THREAD

**Issue by fernandoamz** [aromalanil/useCustomHooks#16]
**Describe the hook**
A hook to get the latitude and longitude

**Additional context**
Create a hook to get the current position from the browser.

```
import React from "react"
import useGeolocalisation from "useGeolocalisation"

const Home = () => {
  /*
  * The hook will return you an object like this: 
  * 
  * {
  *   latitude: 1233234,
  *   longitude: -1234345,
  * }
  * 
  */

  const [geolocalitation] = useGeolocalisation(true);

  return (
    <div>
      {geolocalitation.latitude}
      {geolocalitation.longitude}
    </div>
  );
}
```

**Comment by sairaj2119** [comment]
Hey @fernandoamz  assign me I already wrote code for that

**Comment by fernandoamz** [comment]
@sairaj2119, This hook is written now https://github.com/aromalanil/useCustomHooks/pull/15

**Comment by aromalanil** [comment]
@sairaj2119 Thanks for your interest, @fernandoamz is working on this hook now. We value your contribution, so please consider adding another hook.

**Comment by sairaj2119** [comment]
Common dude I already made a pull request

**Comment by sairaj2119** [comment]
I also followed the documentation style and comments check it and let me know what you think about the code

**Comment by fernandoamz** [comment]
@sairaj2119, I commented you here https://github.com/aromalanil/useCustomHooks/issues/16#issuecomment-702362425, that I sended a pull request for this hook

**Comment by sairaj2119** [comment]
So what can I do,

**Comment by aromalanil** [comment]
@sairaj2119 You can always contribute other hooks. If you have a suggestion for a hook. Please raise an issue, and mention if you want to work on it.

**Comment by fernandoamz** [comment]
@sairaj2119, You can suggest a hook to detect if the user is using a mobile or browser.

**Comment by aromalanil** [comment]
Solved in #15
